test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Csv.Tests.ps1

```diff
@@ -71,3 +19,85 @@ Describe "Import-Csv DRT Unit Tests" -Tags "CI" {
         $returnObject.Second | Should Be 2
     }
 }
+
+Describe "Import-Csv File Format Tests" -Tags "CI" {
+    BeforeAll {
+        # The file is w/o header
+        $testImportCsv1 = Join-Path -Path (Join-Path $PSScriptRoot -ChildPath assets) -ChildPath TestImportCsv1.csv
+        # The file is with header
+        $testImportCsv2 = Join-Path -Path (Join-Path $PSScriptRoot -ChildPath assets) -ChildPath TestImportCsv2.csv
+        # The file is W3C Extended Log File Format
+        $testImportCsv3 = Join-Path -Path (Join-Path $PSScriptRoot -ChildPath assets) -ChildPath TestImportCsv3.csv
+        
+        $testCSVfiles = $testImportCsv1, $testImportCsv2, $testImportCsv3
+        $orginalHeader = "Column1","Column2","Column 3"
+        $customHeader = "test1","test2","test3"
+    }
+    # Test set is the same for all file formats
+    foreach ($testCsv in $testCSVfiles) {
+       $FileName = (dir $testCsv).Name
+        Context "Next test file: $FileName" {
+            BeforeAll {
+                if ($FileName -eq "TestImportCsv1.csv") {
+                    # The file does not have header
+                    # (w/o Delimiter here we get throw (bug?)) 
+                    $HeaderParams = @{Header = $orginalHeader; Delimiter = ","}
+                    $CustomHeaderParams = @{Header = $customHeader; Delimiter = ","}
+                } else {
+                    # The files have header
+                    $HeaderParams = @{Delimiter = ","}
+                    $CustomHeaderParams = @{Header = $customHeader; Delimiter = ","}
+                }
+
+            }
+
+            It "Should be able to assign to a variable" {
+                $actual = Import-Csv -Path $testCsv @HeaderParams
```

Comment:
This test is redundant because all the other tests verify the same condition as well. If Import-Csv cannot assign the result to a variable, you won't be able to validate that it imported the correct headers and values